CMakeLists.txt

```diff
@@ -133,17 +133,9 @@ option(
     OFF
 )
 
+# Whether the libavif library uses c++ indirectly (e.g. through linking to libgav1).
```

Comment:
I suggest we rename the variable to reflect the new, more specific meaning. The new name should have "LIB" in the name, e.g., "AVIF_LIB_USE_CXX" or "AVIF_USE_CXX_IN _LIB" or "AVIF_LINK_LIB_AS_CXX".

Reply:
AVIF_LIB_USE_CXX ssems like the best one to me